File: app/src/main/java/com/garymcgowan/moviepedia/network/OmdbSearch.java
package com.garymcgowan.moviepedia.network;

import com.garymcgowan.moviepedia.model.Movie;
import com.google.gson.annotations.SerializedName;

import java.util.List;

/**
 * Created by Gary on 2016/10/26.
 */

public class OmdbSearch extends OmdbBaseResponse {


    @SerializedName("Search") public List<Movie> search;
    public int totalResults;

}

File: app/src/main/java/com/garymcgowan/moviepedia/network/OmdbSearch.kt
package com.garymcgowan.moviepedia.network

import com.garymcgowan.moviepedia.model.Movie
import com.google.gson.annotations.SerializedName

class OmdbSearch : OmdbBaseResponse() {

    @SerializedName("Search") var search: List<Movie>? = null
    @SerializedName("totalResults") var totalResults: Int = 0

}
